osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -75,6 +76,8 @@ public enum GlobalAction
         [Description("Quick Retry (Hold)")]
         QuickRetry,
 
+        [Description("Toggle gameplay mouse buttons")]
+        ToggleGameplayMouseButtons,
```

Comment:
due to the way database storage works, this will cause the shortcut from screenshots to be transferred here (they are using the int representations) so this needs to be the last entry in the enum.

Reply:
Ah dang, you're right.